43.md

```diff
@@ -0,0 +1,191 @@
+NIP-43
+======
+
+Private DM
+----------
+
+`draft` `optional`
+
+This direct message (DM) scheme between two participants is end-to-end encrypted, hides
+most metadata, has forward secrecy and aims to minimize spam.
+
+This NIP currently uses [NIP-44](44.md) v2 encryption.
+
+## Flow
+
+- Before sending a DM to someone (`B`), an user (`A`) picks a random privkey to be used as chat session (`session-privkey-A`);
```

Comment:
```suggestion
- Before sending a DM to someone (`B`), an user (`A`) picks a random privkey to be used as chat session (`session-privkey`);
```

Will there be a `session-privkey-B`? `A` to `B` and `B` to `A` are the same session, aren't they?

Reply:
Yeah it could confuse people. There is no session-privkey-B. They use the same privkey.
Didn't notice in time I could've applied yout commit suggestion sorry. Renamed other occurrences too